### ISSUE TITLE
Apply dot graph layouts

### ISSUE THREAD

**Issue by bit-pirate** [bponsler/roslaunch_to_dot#9]
For bigger launchers the dot graph becomes extremely wide making it hard to read.

If there are options to modify/control/steer the layout, it would be good to make the graphs more narrow, but longer (e.g. for printing, documentation etc.).

**Comment by bponsler** [comment]
Thanks for the suggestion, I'll look into it and see what can be done to improve this.

**Comment by bponsler** [comment]
I just added two new command line options (see commit 3a60d0c63882e8a2ed8d52248739b66bdc89d554):

```
--landscape -- this will display the nodes from left to right instead of top to bottom
--aspect-ratio -- this will set the desired aspect ratio for the graph. The default is 8.5/11
```

After I enabled these the resulting graph seemed easier to view (granted my launch trees aren't very enormous). I'm not sure these options will improve your graphs enough to view/print but hopefully they help.

Let me know if it's still really hard to view and I'll see about other adjustments I can make.
